src/main/java/com/rogerr/custom/model/Login.java

```diff
@@ -0,0 +1,30 @@
+package com.rogerr.custom.model;
+
+import javax.validation.constraints.NotBlank;
+
+public class Login {
+	
+	
+	@NotBlank(message="username can NOT be empty!")
```

Comment:
ATM I would avoid using Java Validation on your objects, can cause A LOT of hang-ups/slowdowns when you're getting started stick to field-level validations in the views themselves.  If it's just this, that's fine, but be careful.